test/test-applications/integrations/Samples.Dapper/Samples.Dapper.csproj

```diff
@@ -3,8 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
      <!-- override to remove net452 -->
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net461;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
-    <TargetFrameworks Condition="$(MSBuildVersion) >= 16.8.0 AND !$(TargetFrameworks.Contains('net5.0'))">$(TargetFrameworks);net5.0</TargetFrameworks>
+    <TargetFrameworks>net461;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
https://github.com/DataDog/dd-trace-dotnet/pull/1477 is about _not_ building samples unless we're going to use them in CI (to avoid waste). This change (and other samples changes) make these build in CI on Linux where previously they weren't. Is that a change we def want to make? 

FWIW, I'd rather have the simpler project files, as we can do optimisations in the CI build itself for speed separately.

Reply:
TBH, #1477 was more about getting around build errors in one of those projects (i.e. kicking the can down the road 😅). Adding an extra target here on Linux/macOS only adds a few seconds to the build, and I'd rather have the simpler project files as well. It may even help Linux/macOS users find compiler errors that only happen when targeting .NET Framework.